scripts/export_hf_datasets_to_jsonl.py

```
@@ -0,0 +1,105 @@
+        type=int,
+        default=None,
+        help="Batch size for Dataset.to_json (defaults to HF config default).",
+    )
+    args = parser.parse_args()
+
+    output_dir = Path(args.output_dir)
+    selected = args.dataset or list(DATASETS.keys())
+
+    for key in selected:
+        if key not in DATASETS:
+            raise ValueError(f"Unknown dataset key: {key}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)